ludwig/encoders/sequence_encoders.py

```diff
@@ -1327,8 +1327,7 @@ def __init__(
             weights_initializer=weights_initializer,
             recurrent_initializer=recurrent_initializer,
             bias_initializer=bias_initializer,
-            dropout=dropout,
-            recurrent_dropout=recurrent_dropout,
+            dropout=recurrent_dropout,
```

Comment:
The docstring of this class is also missing an entry for `recurrent_dropout`. I guess I could imagine a case for punting the description of recurrent_dropout downstream into the RecurrentStack class– just wanted to call it out. Its just weird that there are two dropouts, so it could be good as a user to be able to read up on the difference without inspecting the code too closely.